testkit/src/main/scala/pureconfig/arbitrary/package.scala

```diff
@@ -1,31 +1,41 @@
 package pureconfig
 
+import java.io.File
+import java.math.BigInteger
+import java.nio.file.Path
+import java.time
+import java.time._
```

Comment:
nit: this is probably clearer than `time.Duration` to distinguish it from Scala `Duration`.

```suggestion
import java.time.{Duration => JavaDuration, _}
```